src/System.Private.ServiceModel/tests/Common/Infrastructure/src/BridgeClientCertificateManager.cs

```diff
@@ -176,10 +176,13 @@ public static void InstallLocalCertificateFromBridge()
                 }
             }
 
-            InstallCertificateToMyStore(certificateToInstall);
-
-            // We also need to install the root cert if we install a local cert
-            InstallRootCertificateFromBridge();
+            // certificateToInstall could be null if local cert has been installed in the previous tests
+            if (certificateToInstall != null)
+            {
```

Comment:
This comment and test makes me nervous.  Tests can run in parallel.  Can more than one set this value?  What happens if 2 tests run concurrently and change this value?  Can one test be in the middle of this != null test and have another test change it?  If so, we need to make this thread safe with a lock, or we need tests to pass in this information as they need it.